### PR TITLE
GitHubSync update - release-9.0

### PR DESCRIPTION
**Pull request by particularbot6**
This is an automated synchronization PR.

The following source template repositories were used:
* particular/RepoStandards

### CODE REVIEW

File: src/AcceptanceTests/Sending/When_dispatching_fails.cs [hunk 1]
```diff
@@ -46,7 +46,8 @@ static async Task Send(IMessageSession messageSession)
             await messageSession.Send(new MyMessage()).ConfigureAwait(false);
 
             // https://github.com/Azure/azure-storage-net/issues/534
-            EventHandler<RequestEventArgs> failRequests = delegate { throw new Exception("Fail on proxy"); };
+            EventHandler<RequestEventArgs> failRequests = delegate
+            { throw new Exception("Fail on proxy"); };
```
Comment (bording) [inline src/AcceptanceTests/Sending/When_dispatching_fails.cs:50]: Why did this change?

File: src/Transport/DelayDelivery/SimpleJSON.cs [hunk 2]
```diff
@@ -68,14 +72,12 @@ namespace SimpleJson
...
     using System.Collections.Generic;
     using System.ComponentModel;
-    using System.Diagnostics.CodeAnalysis;
 
     /// <summary>
     /// Represents the json array.
     /// </summary>
     [GeneratedCode("simple-json", "1.0.0")]
     [EditorBrowsable(EditorBrowsableState.Never)]
-    [SuppressMessage("Microsoft.Naming", "CA1710:IdentifiersShouldHaveCorrectSuffix")]
```
Comment (bording) [inline src/Transport/DelayDelivery/SimpleJSON.cs:78]: Same comment about SimpleJson as the `master` PR. Let's revert the changes here.

**Review by bording — CHANGES_REQUESTED** [review]

File: src/AcceptanceTests/Sending/When_dispatching_fails.cs (hunk 1 above)
Comment (SeanFeldman) [reply src/AcceptanceTests/Sending/When_dispatching_fails.cs:50]: Fantastic question. Changes were applied to the solution so not sure why this specific line has changed. I can revert it.

File: src/Transport/DelayDelivery/SimpleJSON.g.cs [hunk 3]
```diff
@@ -50,6 +50,8 @@
 #define SIMPLE_JSON_TYPEINFO
 #endif
 
+#pragma warning disable IDE0065 // Misplaced using directive
```
Comment (bording) [inline src/Transport/DelayDelivery/SimpleJSON.g.cs:53]: You just renamed the file, you didn't revert the changes to it.

File: src/Transport/DelayDelivery/SimpleJSON.cs (hunk 2 above)
Comment (SeanFeldman) [reply src/Transport/DelayDelivery/SimpleJSON.cs]: Done

File: src/AcceptanceTests/Sending/When_dispatching_fails.cs (hunk 1 above)
Comment (SeanFeldman) [reply src/AcceptanceTests/Sending/When_dispatching_fails.cs]: BTW, this is why it was changed: https://builds.particular.net/buildConfiguration/NServiceBus_Transports_NServiceBusAzureStorageQueues_1Compile/706894?

**Review by bording — APPROVED** [review]

**Pull request merged**